templates/zerver/integrations.html

```diff
@@ -533,6 +538,33 @@
       <p><small>Thanks to Wes of TurboVote for <a href="https://gist.github.com/cap10morgan/5100822">submitting this integration</a>!</small></p>
     </div>
 
+    <div id="circleci" class="integration-instructions">
+
+      <p>Zulip supports integration with CircleCi and can notify you of
+          your build statuses.</p>
+
+      <p>First, create the stream you'd like to use for CircleCi notifications,
+          and subscribe all interested parties to this stream. We
+          recommend the name <code>circleci</code>.</p>
+
+      <p><code>{{ external_api_uri }}/v1/external/circleci?api_key=abcdefgh&amp;stream=circleci</code></p>
+
+      <p>where <code>api_key</code> is the API key of your Zulip bot,
+      and <code>stream</code> is the stream name you want the
+      notifications sent to.</p>
+
+      <p>
+        Next, modify your <code>circle.yml</code> like described
+        <a href="https://circleci.com/docs/configuration/#notify">here</a>.
+      </p>
+
+      <p><b>Congratulations! You're done!</b><br />  When a build is done,
+          you'll get a notification like this:</p>
+
+      <img class="screenshot" src="/static/images/integrations/circleci/001.png" />
```

Comment:
for the screenshot, can your retake it with the bot's avatar set to the CircleCI logo and sending to a stream called "circleci"?